### DISCUSSION TITLE
XFER Numpad Not Working

### DISCUSSION THREAD

**Discussion by russellmcg** [nguyenquyhy/Flight-Tracker-StreamDeck#412] [Q&A]
I ended up deleting my numbers 8 and 9 under the XDPR, and I am not sure how to get them to show back up. Any suggestions? I have uninstalled Flight Tracker and deleted the profiles and it still shows up that way.

**Comment by nguyenquyhy** [comment]
Try to delete Numpad profile and click COM/NAV button again. It should ask you to create Numpad profile again.

**Reply by russellmcg** [reply]
Tried that, it still installs it missing the numbers. Here is a video of it.
https://www.dropbox.com/scl/fi/8asj60hj82g4r0tvbrd1z/IMG_0279.MOV?rlkey=bxbct0eunr1fom4vqmx1ajer8&dl=0

**Reply by nguyenquyhy** [reply]
Oh it looks correct to me. Transponder has only digit 0-7.

**Reply by russellmcg** [reply]
Ah! I was not thinking that there was only 7 for it. Thanks!
